src/components/Calculator.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
```

Comment:
[OPTIONAL]
- [ ] You may also `import React, {Component}` and then when creating a class component you only do:
`class Calculator extends Component {}`